src/goats_cli/processes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Enums for known subprocesses managed by GOATS CLI.
+"""
+
+__all__ = ["ProcessName"]
+
+from enum import Enum
+
+
+class ProcessName(str, Enum):
```

Comment:
I switched to an `Enum` as I kept spelling a name wrong, and introduced a bug that got me held up for 30 minutes.